jans-cedarling/bindings/cedarling_python/examples/_bootstrap.py

```
@@ -0,0 +1,46 @@
+_BINDINGS_DIR = os.path.dirname(_EXAMPLES_DIR)
+
+# Ensure the cedarling_python package is importable when running
+# scripts from any working directory (e.g. python examples/unsigned_authz.py).
+if _BINDINGS_DIR not in sys.path:
+    sys.path.insert(0, _BINDINGS_DIR)
+
+from cedarling_python import BootstrapConfig  # noqa: E402
+EXAMPLE_FILES_DIR = os.path.join(_BINDINGS_DIR, "example_files")
+
+
+def make_bootstrap_config(policy_store_path, extra_config=None):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `make_bootstrap_config`

(ANN202)